packages/CodeQuality/src/Rector/FuncCall/SimplifyArrayCallableRector.php

```diff
@@ -25,6 +25,12 @@
     private $functionsWithCallableArgumentPosition = [
         'array_filter' => 1,
         'array_map' => 0,
+        'array_walk' => 1,
+        'array_reduce' => 1,
+        'usort' => 1,
+        'uksort' => 1,
+        'uasort' => 1,
+        'array_walk_recursive' => 1,
```

Comment:
We should list all of them. I'll provide a full list later

Reply:
I tried to Google them, but dunno how to find them all. 

Thanks :+1: